src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -162,6 +162,9 @@
              :n {:title "line numbers"
                  :target (fn [atom] (.getView (.-views atom) (.getActiveTextEditor (.-workspace atom))))
                  :action "editor:toggle-line-numbers"}
+             :h {:category "highlight"
+                 :h {:title "highlight current line"
```

Comment:
Hmm I think with only 1 highlighting item we shouldn't make command chains too long. Do you think you could find a different keybinding under `t` you would be happy with?